toolkits/Dockerfile.ml

```
@@ -296,4 +299,6 @@ ARG VERSION=0.0.0
 LABEL org.opencontainers.image.title="AICA Machine Learning Tool`kit"
 LABEL org.opencontainers.image.description="AICA Machine Learning Toolkit (GPU support)"
 LABEL org.opencontainers.image.version="${VERSION}"
```

I think this should be
```suggestion
LABEL tech.aica.image.metadata='{"type":"lib","dependencies":{\
```

Packages means: "this package also provides the following". This is mostly used by `core` to provide CL and modulo.

Dependencies means: "this package needs those installed as well".

You can check how this is used in our stack on the Registry.

In the case where this should be a "dependency", it should also contain a constraint instead of a version (again the Registry will show you examples), I am imagine you would want something like `~> v0.1.0`.